src/apply.jl

```diff
@@ -39,22 +39,36 @@ end # loop
 # TimeArray <--> Scalar
 for op in [MATH_ALL; COMPARE_DOTONLY]
     @eval begin
-        function ($op){T<:Number,N}(ta::TimeArray{T,N}, var::Number)
+        function broadcast(::typeof($op), ta::TimeArray{T, N},
+                           var::Number) where {T <: Number, N}
             cnames  = [name * string($op) * string(var) for name in ta.colnames]
-            vals = ($op)(ta.values, var)
+            vals = broadcast($op, ta.values, var)
             TimeArray(ta.timestamp, vals, cnames, ta.meta)
         end # function
+
+        # FIXME: make some function allowed dot version only
+        # e.g. `ta > 42` is not allowed, but `ta .> 42` does
+        if Symbol($op) ∉ [$MATH_DOTONLY; $COMPARE_DOTONLY]
+            ($op)(ta::TimeArray{T, N}, var::Number) where {T <: Number, N} =
```

Comment:
In this commit (9237607), 
  - `cl .+ 42` works
  - `cl + 42` works
  - `cl > 0` raise error, this is expected
  - `cl .> 0` raise error, it should broadcast
```julia
julia> cl .> 0
ERROR: MethodError: no method matching isless(::Int64, ::TimeSeries.TimeArray{Float64,1,Date,Array{Float64,1}})
Closest candidates are:
  isless(::Real, ::AbstractFloat) at operators.jl:97
  isless(::Real, ::Real) at operators.jl:266
Stacktrace:
 [1] (::##5#6)(::TimeSeries.TimeArray{Float64,1,Date,Array{Float64,1}}) at ./<missing>:0
 [2] broadcast(::Function, ::TimeSeries.TimeArray{Float64,1,Date,Array{Float64,1}}) at ./broadcast.jl:434
```

any idea?